validator/client/validator.go

```diff
@@ -566,7 +563,7 @@ func (v *validator) UpdateDuties(ctx context.Context, slot primitives.Slot) erro
 
 	v.dutiesLock.Lock()
 	v.duties = resp
-	v.logDuties(slot, v.duties.CurrentEpochDuties, v.duties.NextEpochDuties)
+	v.logDuties(ss-params.BeaconConfig().SlotsPerEpoch, v.duties.CurrentEpochDuties, v.duties.NextEpochDuties)
```

Comment:
Can't this underflow for kurtosis?

Reply:
no, `ss` is defined to be the start of the next epoch, therefore it is always at least `SlotsPerEpoch`